integration-tests/api-gateway/check/check-happy.test.ts

```diff
@@ -60,4 +60,30 @@ describe("Given the session and NINO is valid", () => {
     const checkData = check.status;
     expect(checkData).toEqual(200);
   });
+
+  it("should 400 when provided with JS in the session header", async () => {
```

Comment:
should this be 500 instead of 400 as it's asserting on 500 status code?

Reply:
Good spot, thanks I have updated